templates/js-cypress/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cypress-module-api",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "Run Cypress via its module API",
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "dependencies": {
+    "cypress": "^12.7.0",
```

Comment:
I'm not sure we need to include cypress if it's already in the base image, we should test this.

Reply:
moving cypress to devDependencies resulted in following error (see the package.json prnt scrn and error prnt scrn) 🤔  moving it back to dependencies for now

<img width="380" alt="Screenshot 2023-03-10 at 16 21 26" src="https://user-images.githubusercontent.com/56041262/224354920-4a4a7264-2520-4376-9cfd-289f84a73228.png">

<img width="992" alt="Screenshot 2023-03-10 at 16 22 58" src="https://user-images.githubusercontent.com/56041262/224354975-5ce4808f-b59d-4831-a4a1-abc565dc1cd8.png">